package.json

```diff
@@ -49,15 +54,15 @@
     "bin": {
         "minilogo-cli": "./bin/minilogo.js"
     },
-    "exports": "./out/extension.js",
-    "main": "./out/extension.js",
+    "exports": "./out/extension.cjs",
```

Comment:
Actually I would really like to switch to ESM at some point. The cjs setup is more legacy anyways, I just never made the migration. But that can be done later after we're on Langium 3.0